Samu/enhancement/428-update-members-page

SProkopios opened the pull request:
Closes #428

## 🔧 **Changes Made**

- Updated Team namings in mapper
- Created a Navbar for This Page
- Implemented New Figma Stylings
- Added new api for teams

## ✅ **Checklist Before Submission**

- **Functionality**: I have tested my code, and it works as expected.
- **JSDoc**: I have added or updated JSDoc comments for all relevant code.
- **Debugging**: No `console.log()` or other debugging statements are left.
- **Clean Code**: Removed commented-out or unnecessary code.
- **Tests**: Added new tests or updated existing ones for the changes made.
- **Documentation**: Documentation has been updated (if applicable).

---

## 📝 **Additional Information**

Provide any additional context or information that reviewers may need to know:

- Teams and members will be modified next in directus, mapper already is set for new teams

Jonroi commented on frontend-next-migration/src/app/[lng]/(intro)/team/Layout.module.scss:
```diff
@@ -1,14 +1,5 @@
-}
-
-
-.diveButton{
```
memo: reattach

Jonroi reviewed (changes requested):
The following adjustments are required. These items have been discussed previously, so no further instructions are needed:

- Sidebar gap/spacing.
- Improve responsiveness across various sections to better reflect the Figma designs.
- Restore the `DiveButton` component.
- Implement SCSS breakpoints for consistent responsive behavior.

SProkopios commented:
- Refactored to follow feature-slice desing (moving lot of components from pages to layout)
- Implemented SCSS breakpoints instead of using useSizes. (as much as i can)
- Restored the DiveButton component.
- Improved responsiveness across various sections to better reflect the Figma designs.
- Sidebar gap/spacing.
- code cleanup in teamheader

Jonroi reviewed (approved):
This is solid work. Since we understand your situation, I won’t list the minor observations here. Great job overall thank you for your effort and dedication!
